packages/custom-functions-metadata/Specification.md

```diff
@@ -39,6 +40,22 @@ If the last function parameter is of type `CustomFunctions.CancelableInvocation`
 Indicates that the custom function captures the calling object i.e. the [Excel.CellValue](https://learn.microsoft.com/es-es/javascript/api/excel/excel.cellvalue)
 object that invoked the custom function, in the first parameter.
 
+---
+### @customEnum
```

Comment:
Do we need to specify it's introducing a new usage, same layer tag with @customfunction, and other tags couldn't decorate this tag?

Reply:
Good point. I should and will separate the `@customEnum` with the tags for functions.